CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## unreleased
 * Remove `iosBaseSDK`, `iosDeploymentTarget`, `iosIdentifierForVendor`, `deviceAppGeneratedPersistentUuid`, and `deviceScreenOrientation` from `BTAnalyticsMetadata`
+* Breaking Changes
+  * BraintreePaymentFlow
+    * Replaced deprecated `SFAuthenticationSession` with `ASWebAuthenticationSession` in payment flow
+    * Removed `BTViewControllerPresentingDelegate` protocol and correlating methods
+    * Removed `load` method from `BTPaymentFlowClient`
```

Comment:
Don't think this needs a CHANGELOG, since not a public method

Reply:
Cool - wasn't sure since I know we've had merchants comment on compile times with the load method previously (I think we had a GH issue for the one in PayPalDriver at one point) if it was worth calling out.